packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -26,6 +26,15 @@
  * @prop --calcite-panel-content-space: [Deprecated] Use `--calcite-panel-space` instead. Specifies the padding of the component's content.
  * @prop --calcite-panel-footer-padding: [Deprecated] Use `--calcite-panel-footer-space` instead. Specifies the padding of the component's footer.
  * @prop --calcite-panel-header-border-block-end: [Deprecated] Use `--calcite-panel-border-color` instead. Specifies the component header's block end border.
+
+ * @prop --calcite-panel-header-action-menu-items-space: Specifies the space between actions in the header's action menu.
+ * @prop --calcite-panel-header-action-background-color-hover: Specifies the background color of the component's `calcite-action` items  in the panel header when hovered.
```

Comment:
Looks like a double space here inbetween `items  in`. Applies to multiple lines here.